jobs/SDE/Assignments/SDE-FE-Assignment.md

```diff
@@ -0,0 +1,76 @@
+### SDE coding challenge
+
+Here you are! We're happy that you're interested in working for Quantum Inventions! 
+To get a better idea of your current skills, we'd like you to complete a coding challenge - build a web app to display and update resources (e.g - users). 
+The actual requirements are listed further down, but here are the general tech requirements:
+- Make sure your app is runnable on Windows, Mac OS X and Linux
+- Do not include any paid software dependencies
+- Include instructions for setting up and running your application
+- Containerized app is a plus
+
+### Mock APIs 
+
+Simulate API like behaviour using following approach -
+* You can use API mock service: https://reqres.in/. Visit the site for more details.
```

Comment:
This should be left open-ended as just one of the generic guidelines. The example service is also not required.

Reply:
I am actually keen to enforce to use mock service because it serves all the necessary API and related flavours (pagination, delayed response) that is required for our assignment. If we omit this, we believe for a candidate, it might become a bit confusing which process to choose( and that might not support all the functionalities we expect at the end).

Reply:
We could put this point as a guideline in the beginning summary (in the generic do's and dont's). But still avoid naming any specific service.

Reply:
>We could put this point as a guideline in the beginning summary
Ok, Noted

If we avoid naming any specific service, the application needs might not fully suffice by the approach candidate choose. I feel, from FE perspective, it's more general to see how the candidate handles the ajax requests rather than which BE Mock API service they choose.